Author: Denis Kristak (Inuits)
Remove `dev` deps to avoid using unsupported `group` keyword (and we don't need them anyway).
diff -ruN dataclasses_json-0.6.1_orig/pyproject.toml dataclasses_json-0.6.1/pyproject.toml
--- dataclasses_json-0.6.1_orig/pyproject.toml	2023-09-26 14:00:32.000000000 +0100
+++ dataclasses_json-0.6.1/pyproject.toml	2023-10-16 16:19:43.666996763 +0100
@@ -13,25 +13,13 @@
 typing-inspect = ">=0.4.0, <1"
 marshmallow = ">=3.18.0,<4.0.0"
 
-[tool.poetry.group.dev.dependencies]
-pytest = "^7.1"
-pytest-cov = "~2.12"
-mypy = ">=0.710, <2"
-hypothesis = "^6.77"
-portray = "^1.6"
-flake8 = "^5"
-simplejson = "^3.19.1"
-black = "^22"
-
 [tool.black]
 line-length = 120
 
 [build-system]
-requires = ["poetry-core>=1.2.0", "poetry-dynamic-versioning"]
-build-backend = "poetry_dynamic_versioning.backend"
+requires = ["poetry-core>=1.2.0"]
+build-backend = "poetry.core.masonry.api"
 
-[tool.poetry-dynamic-versioning]
-enable = true
 
 [tool.poetry.urls]
 changelog = "https://github.com/lidatong/dataclasses-json/releases"
